### DISCUSSION TITLE
Past Chatlogs

### DISCUSSION THREAD

**Discussion by podkilla** [Chatterino/chatterino2#5523] [Q&A] [answered]
When i start chatterino it loads the chatlog from the channels, even if i haven't been online and there has been a stream that day the chatlog is shown in chatterino. Why is that chatlog not written to the channel log file and is there a way to log past chat from that day to the logfile.

**Comment by Felanbird** [comment, marked as answer]
The local channel logs are for messages you saw live, if we were to populate that file every time you opened the client and loaded message history, it would likely just add a sea of dupe messages in many of the non-active channels. 
To prevent that dupe message issue we'd probably have to check if the most recent messages in history are already logged, but if the message history has 600 messages, and 2 people have a 20 message conversation, it's likely the client would think the message history doesn't match up and log all the messages again. So then we have to think about how much of the logs the client should read to prevent that.
